resources/views/product/overview.blade.php

```diff
@@ -25,13 +25,13 @@
                     @endif
                     <div>
                         <div class="border-t pt-5 text-lg font-bold">@lang('Description')</div>
-                        <div class="prose text-inactive" itemprop="description">
+                        <div class="prose text-muted" itemprop="description">
                             {!! $product->description !!}
                         </div>
                     </div>
                     <div>
                         <div class="mb-2 border-t pt-5 text-lg font-bold">@lang('Specifications')</div>
-                        <dl class="flex flex-col text-inactive [&>dd]:rounded [&>dd]:p-2 odd:[&>dd]:bg-highlight odd:[&>dd]:font-semibold odd:[&>dd]:text-neutral even:[&>dd]:pl-4">
+                        <dl class="flex flex-col text-muted [&>dd]:rounded [&>dd]:p-2 odd:[&>dd]:bg odd:[&>dd]:font-semibold odd:[&>dd]:text even:[&>dd]:pl-4">
```

Comment:
this can be *: since everything is a <dd

Reply:
A bit out of scope for this PR, but every improvement is welcome. Who want to PR this?

Reply:
Seems like that happend here: https://github.com/rapidez/core/pull/307/files#diff-99969e2a5bbf40165a88c6e09d3d7644a5f2c99598dfca5fbe042d178542a5cf, could you fix that @JimmyHoenderdaal? It should stay a dl with dt's and dd's. Please double check the styling! Wrapping the dt+dd within a div is allowed as also used within the cart/checkout sidebar.

Reply:
Internal ref: RAP-1041